app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Api::V1::ReviewsController < ApplicationController
+
+  def create
+    binding.pry
+    # look at params, get show object so that review can belong to a show
+    new_review = Review.new(review_params)
+
```

Comment:
in order to make this happen, you have to have a nested route to this create action, just like last week, but with api/vi in it.